test/integration/util.go

```diff
@@ -57,5 +57,5 @@ func UniqueProfileName(prefix string) string {
 		return "minikube"
 	}
 	// example: prefix-20200413T162239-3215
-	return fmt.Sprintf("%s-%s-%d", prefix, time.Now().Format("20060102T150405"), os.Getpid())
+	return fmt.Sprintf("%s-%s-%d", prefix, time.Now().Format("20060102150405"), os.Getpid())
```

Comment:
Can this be part of another PR? I don't understand its relevance.

Reply:
the multinode test that I added fails without this change

Reply:
do u mind sharing the failed message in an issue and add in the PR description, that this fixes that ?

Reply:
there's no real message, kubeadm join just fails with exit code 70

Reply:
added

Reply:
@sharifelgamal can u update the comment with the right example

Reply:
example containerd-20200417104139-6189

Reply:
Fixed